src/deluge/dsp/compressor/rms_feedback.h

```diff
@@ -91,8 +91,8 @@ class RMSFeedbackCompressor {
 
 private:
 	// parameters in use
-	float a_ = (-1000.0 / 44100);
-	float r_ = (-1000.0 / 44100);
+	float a_ = (-1000.0 / kSampleRate);
```

Comment:
oops - these should be -1000.0f if you don't mind changing it, I think they're doubles right now

Reply:
Fixed!